### PR TITLE
dom-0: xen config 'device_tree' dynamic filling

### PR DESCRIPTION
**Pull request by dterletskiy**
Implement dynamic filling 'device_tree' parameter in xen configuration files for dom-u and dom-a depending on build configuration.

### CODE REVIEW

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend [hunk 1]
```diff
@@ -43,13 +47,14 @@ do_install:append() {
     # Install domu-set-root script
     install -d ${D}${libdir}/xen/bin
     install -m 0744 ${WORKDIR}/domu-set-root ${D}${libdir}/xen/bin
-    install -d ${D}${sysconfdir}/systemd/system/domu.service.d
```
Comment (lorc) [inline meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:46]: This is not related to device trees. Please move those changes into a separate commit

**Review by lorc — CHANGES_REQUESTED** [review]

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend (hunk 1 above)
Comment (dterletskiy) [reply meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:46]: Agree.
Done.

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by dterletskiy** [comment]
Done
